components/PlantForm/ImageUpload.js

```diff
@@ -0,0 +1,9 @@
+export default function ImageUpload() {
+  return (
+    <>
+      <label htmlFor="image-upload">
+        <input name="imageUrl" id="image-upload" type="file" accept="image/*" />
+      </label>
+    </>
```

Comment:
You dont need empty tags here :)

```suggestion
      <label htmlFor="image-upload">
        <input name="imageUrl" id="image-upload" type="file" accept="image/*" />
      </label>
```